jenkins-template.yml

```diff
@@ -75,6 +75,38 @@ objects:
 - apiVersion: v1
   kind: ImageStream
   metadata:
+    name: ${JENKINS_SERVICE_NAME}-slave-ubuntu
+    annotations:
+        slave-label: ubuntu
+    labels:
+      role: jenkins-slave
+- apiVersion: v1
+  kind: BuildConfig
+  metadata:
+    name: ${JENKINS_SERVICE_NAME}-slave-ubuntu
+  spec:
+    output:
+      to:
+        kind: ImageStreamTag
+        name: ${JENKINS_SERVICE_NAME}-slave-ubuntu:latest
+    runPolicy: Serial
+    source:
+      git:
+        uri: https://github.com/adamsaleh/wendy-jenkins-s2i-configuration
+        ref: slave-ubuntu
+      contextDir: slave-bob-ubuntu
```

Comment:
Is this a sensible name for this? Do we want to mention bob here? Going by how we have done the rest, the name has matched the directory the Dockerfile is in, would be nice to keep this consistent. (slave-ubuntu)